rules/prefer-export-from.js

```diff
@@ -122,20 +124,22 @@ function getFixFunction({
 				? exported.text
 				: `${imported.text} as ${exported.text}`;
 
-			if (exportDeclaration) {
+			const specifierWithKind = isTypeExport ? `type ${specifier}` : specifier;
```

Comment:
This should not work like this, only if existing export is value export, add `type `, but we should also be able to result `export type {} from ''`, in this case `type ` is not needed.

Reply:
Now existing export is always a value export